probes/http/http.go

```diff
@@ -552,7 +552,7 @@ func (p *Probe) Start(ctx context.Context, dataChan chan *metrics.EventMetrics)
 			break
 		}
 		p.updateTargetsAndStartProbes(ctx, dataChan)
-		time.Sleep(p.opts.Interval)
+		time.Sleep(time.Second)
```

Comment:
I think it's better to do whatever is smaller, e.g. if p.opts.Interval is 10ms. What do you think?

initialTargetRefreshPeriod := p.opts.Interval
if initialTargetRefreshPeriod > time.Second {
  initialTargetRefreshPeriod = time.Second
}